machine_management/user_infra/adding-compute-user-infra-general.adoc

```diff
@@ -0,0 +1,32 @@
+[id="adding-compute-user-infra-general"]
+= Adding compute machines to clusters with user-provisioned infrastructure
+include::modules/common-attributes.adoc[]
+:context: adding-compute-user-infra-general
+
+toc::[]
+
+In addition to adding compute machines to a cluster during the user-provisioned infrastructure installation process, you can add more compute machines to a cluster with user-provisioned infrastructure. The process requires some of the same configuration files and parameters used during installation.
+
+== Adding compute machines to Amazon Web Services
+
+To add more compute machines to your {product-title} cluster on Amazon Web Services (AWS), see xref:../../machine_management/user_infra/adding-aws-compute-user-infra.adoc#adding-aws-compute-user-infra[Adding compute machines to AWS by using CloudFormation templates].
+
+== Adding compute machines to Microsoft Azure
+
+To add more compute machines to your {product-title} cluster on Microsoft Azure, see xref:../../installing/installing_azure/installing-azure-user-infra.adoc#installation-creating-azure-control-plane_installing-azure-user-infra[Creating the control plane machines in Azure] in the _Installing a cluster on Azure using ARM templates_ documentation.
```

Comment:
My understanding was that the UPI steps leverage ARM templates, so that's why the procedure is named that way. But I could definitely be incorrect about that! I will verify with Cody later this week.

Reply:
cool, thanks!

Reply:
Hm, is there a reason why we are referencing to Control Plane machines creation instead of workers?

Reply:
Because I grabbed the wrong link :) Fixed now, thanks!

Reply:
Verified "ARM" name with Cody - that's just how we refer to the Azure UPI stuff since those templates are used for a UPI install :)